src/Employer/Employer.Web/Configuration/ConfigurationExtensions.cs

```diff
@@ -88,8 +88,7 @@ public static void AddMvcService(this IServiceCollection services, IHostingEnvir
                 opts.Filters.AddService<PlannedOutageResultFilter>();
 
                 opts.AddTrimModelBinderProvider(loggerFactory);
-            })
-            .AddFluentValidation(fv =>fv.RegisterValidatorsFromAssemblyContaining<Startup>())  
+            })  
```

Comment:
Can you please test if the validations are still working on application reviews?

Reply:
👍 Will do

Reply:
Doh! Still have to configure FluentValidation for MVC. Have now fixed. Good shout!